trieste/acquisition/optimizer.py

```diff
@@ -98,32 +100,54 @@ def automatic_optimizer_selector(
 
     else:
         raise NotImplementedError(
-            f""" No optimizer currentely supports acquisition function
+            f""" No optimizer currently supports acquisition function
                     maximisation over search spaces of type {space}.
                     Try specifying the optimize_random optimizer"""
         )
 
 
-def optimize_discrete(space: DiscreteSearchSpace, target_func: AcquisitionFunction) -> TensorType:
+def optimize_discrete(
+    space: DiscreteSearchSpace,
+    target_func: Union[AcquisitionFunction, Tuple[AcquisitionFunction, int]],
+) -> TensorType:
     """
     An :const:`AcquisitionOptimizer` for :class:'DiscreteSearchSpace' spaces and
     batches of size of 1.
```

Comment:
"batches of size 1" should probably be corrected in the light of the following paragraph